solidity/test/helpers/contract-test-helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { BigNumber } from "@ethersproject/units/node_modules/@ethersproject/bignumber"
+import { ethers } from "hardhat"
+
+export function to1ePrecision(n: number, precision: number): BigNumber {
+  const decimalMultiplier = ethers.BigNumber.from(10).pow(precision)
+  return ethers.BigNumber.from(n).mul(decimalMultiplier)
+}
+
+export function to1e18(n: number): BigNumber {
+  const decimalMultiplier = ethers.BigNumber.from(10).pow(18)
+  return ethers.BigNumber.from(n).mul(decimalMultiplier)
+}
+
+export async function getBlockTime(blockNumber: number): Promise<number> {
+  return (await ethers.provider.getBlock(blockNumber)).timestamp
+}
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
```

Comment:
Not in this PR, but are functions from this file something we can abstract out to https://github.com/keep-network/hardhat-helpers ? Worth opening a PR, to `hardhat-helpers`, we are duplicating it in every repo now. 

cc @nkuba

Reply:
I got rid of it. I suggest we use:
```
const ZERO_ADDRESS = ethers.constants.AddressZero
```
whenever we need a zero address.